<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8" />
	<meta http-equiv="X-UA-Compatible" content="IE=edge" />
	<link rel="icon" href="../assets/img/computer.ico" />
	<meta name="viewport" content="width=device-width, initial-scale=1.0" />
	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/animate.css/4.1.1/animate.min.css" />
	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.min.css" integrity="sha512-MV7K8+y+gLIBoVD59lQIYicR65iaqukzvf/nwasF0nqhPay5w/9lJmVM2hMDcnK1OnMGCdVK+iQrJ7lzPJQd1w==" crossorigin="anonymous" referrerpolicy="no-referrer" />
	<link rel="preconnect" href="https://fonts.googleapis.com" />
	<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
	<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-KK94CHFLLe+nY2dmCWGMq91rCGa5gtU4mk92HdvYe+M/SXH301p5ILy+dN9+nJOZ" crossorigin="anonymous" />
	<link rel="stylesheet" href="https://unpkg.com/aos@2.3.1/dist/aos.css" />
	<link rel="stylesheet" href="../assets/css/project.css" />
	<link rel="stylesheet" href="../assets/css/waves.css" />
	<link rel="stylesheet" href="../assets/css/btns.css" />
	<title>Work 1</title>
</head>
<body>
	<div class="padre">
		<header class="header">
			<div class="menu">
				<div class="menu-left">
					<span class="menu-title">Metagenome Data Analysis</span>
				</div>
				<div class="menu-right">
					<nav class="nav">
						<a href="/index.html#carta"><i class="fa-solid fa-backward"></i>&nbsp;Back</a>
					</nav>
				</div>
			</div>
		</header>
	</div>
	<div class="text">
		<p>During my training as a biologist, I participated in a research project at the <a href="https://cenpat.conicet.gov.ar/laboratorio-de-microbiologia-ambiental-lma/" 
            target="_blank">Environmental Microbiology Laboratory of the National Patagonian Center</a>
			in Puerto Madryn, Chubut, Argentina. In this project, I developed skills in the use of 
			bioinformatics tools and in creating efficient workflows for data analysis. I also gained 
			experience using the R programming language, applying it to tasks such as managing FASTA files, 
			sequence filtering, frequency table generation, statistical analysis, and phylogenetic tree 
			construction. The project focused on studying the ability of microbial communities from polar 
			and subpolar environments to degrade alginates, which are the main component of brown algae. 
			In some places, these algae are invasive, and their degradation can generate biofuels useful 
			at the industrial level. Alginates lyases, the enzymes responsible for breaking down alginates, 
			are present in many microbial communities. However, a large portion of these microbes has not 
			been cultivated in the laboratory, making their study challenging. To address this, we used 
			metagenomic tools that allow for the analysis of a larger proportion of the microbial community. 
			The methodology developed allowed a detailed analysis of the alginate lyases present in polar and 
			subpolar sediments, providing valuable information about their diversity and geographic 
			distribution. The results were presented in 2014 at the 16th European Congress on Biotechnology 
			in Edinburgh (Scotland) and published in 2016 in the journal <a href="https://enviromicro-journals.onlinelibrary.wiley.com/doi/10.1111/1462-2920.13433" target="_blank">Environmental Microbiology</a> (if you are interested can see the paper <a href="/assets/docs/alginolytic_potential.pdf" target="_blank">here</a>). In this <a href="https://github.com/LucianoAD/alginolytic_potential_bacterial_populations" target="_blank">repository</a>, you will find some of the basic scripts I used to manage the sequences. If you don't want to read so much, here is a brief summary of the methodology used for the analysis.</p>
		<div class="methodology-container">
			<button class="toggle-methodology">
                        <span class="arrow">➔</span> Methodology summary
                    </button>
			<div class="methodology-content">

				<p>To achieve the study’s objectives, metagenomic data from 23 coastal sediment samples obtained from different polar and subpolar regions were analyzed. The samples were sequenced using Illumina HiSeq technology, and the data were stored and processed in the IMG/M database under the project "Marine microbial communities from chronically polluted sediments in four geographic locations," identifying a total of 1.4 x 10⁷ protein-coding genes.</p>

				<div class="section-content">
					<h3>1. Download and Filtering of Sequences:</h3>
					<div class="section-content">
						<ul>
							<li>Genomic sequences were downloaded in FASTA format from the IMG/M database, filtered to include only those with a minimum length of 100 amino acids.</li>
							<li>The functional annotation associated with these sequences was also downloaded, which had been previously performed using databases like COG and PFAM.</li>
						</ul>
					</div>

					<h3>2. Identification of Alginates Lyases:</h3>
					<div class="section-content">
						<ul>
							<li>Various search strategies were applied to identify alginate lyase sequences:
								<ul>
									<li>Keyword filtering: Sequences with the annotation "alginate lyase" were searched, although this methodology was strict and limited the results.</li>
									<li>BLASTp in IMG/M: Using reference sequences downloaded from the CAZy database, searches for similar sequences in the metagenomes were conducted.</li>
								</ul>
							</li>
						</ul>
					</div>

					<h3>3. Classification and Frequency Analysis:</h3>
					<div class="section-content">
						<ul>
							<li>The identified sequences were classified by geographic region, allowing the construction of a frequency table.</li>
							<li>The data were normalized, and differences in the number of sequences between regions were evaluated using the PMCMR package in R.</li>
						</ul>
					</div>

					<h3>4. Construction of a Standalone Database:</h3>
					<div class="section-content">
						<ul>
							<li>An independent database was created with alginate lyase sequences from different polysaccharide lyase enzyme families.</li>
							<li>The dbCAN server was used to verify the presence of conserved domains in the identified sequences.</li>
						</ul>
					</div>

					<h3>5. Phylogenetic Analysis:</h3>
					<div class="section-content">
						<ul>
							<li>A subfamily of long alginate lyases (over 500 amino acids) from family PL17, subfamily 2, which were the most abundant in the samples, was selected.</li>
							<li>To explore phylogenetic relationships, MEGA software was used, applying various techniques such as sequence alignment, amino acid substitution models, and bootstrap tests.</li>
						</ul>
					</div>

					<h3>6. Genomic Context Evaluation:</h3>
					<div class="section-content">
						<ul>
							<li>A script was developed in R to handle and organize the sequences in the database, assigning unique identifiers and managing fragmented or homologous sequences.</li>
							<li>Supercontigs containing the sequences of interest were downloaded from IMG/M, and the headers of the 20 sequences closest to each were extracted to evaluate their genomic context.</li>
						</ul>
					</div>
				</div>
			</div>
		</div>
	</div>
	<script src="../assets/js/projects.js"></script>
</body>
</html>